ydb/core/change_exchange/change_exchange.h

```diff
@@ -8,8 +8,34 @@
 
 #include <util/generic/vector.h>
 
+namespace NKikimr {
+
+namespace NDataShard {
+
+class TChangeRecord;
+
+}
+
+namespace NReplication::NService {
+
+class TChangeRecord;
+
+}
```

Comment:
Может лучше так? Хотя бы понятно что к чему относится, а не просто простыня.
```suggestion
namespace NDataShard {
    class TChangeRecord;
}

namespace NReplication::NService {
    class TChangeRecord;
}
```